Postman/Visual Programming.postman_collection.json

```diff
@@ -22,6 +22,44 @@
 				}
 			},
 			"response": []
+		},
+		{
+			"name": "Save IDE Nodes",
+			"request": {
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n\t\"file\": {{file_location}}, \n\t\"nodes\": {\n    \"I/O\": [\n        {\"key\": \"read-csv\", \"name\": \"Read CSV\", \"numPortsIn\": 0, \"color\": \"black\"}\n    ],\n    \"Manipulation\": [\n        {\"key\": \"filter\", \"name\": \"Filter Rows\", \"color\": \"red\"},\n        {\"key\": \"pivot\", \"name\": \"Pivot Table\", \"color\": \"blue\"},\n        {\"key\": \"multi-in\", \"name\": \"Multi-Input Example\", \"numPortsIn\": 3, \"color\": \"green\"}\n    ]\n}\n}",
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{environment}}/workflow/save_nodes",
+					"host": [
+						"{{environment}}"
+					],
+					"path": [
+						"workflow",
+						"save_nodes"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get IDE Nodes",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": ""
```

Comment:
The API endpoint is missing for the 'GET IDE Nodes' route. The parameters also seem to be missing—the file name is defined for the 'save' route, but not for retrieval.

Reply:
Ops sorry, added it to the collection.